packages/components/chip/src/Chip.stories.tsx

```diff
@@ -7,11 +8,60 @@ const meta: Meta<typeof Chip> = {
   component: Chip,
 }
 
+type ChipProps = ComponentProps<typeof Chip>
+
+const designs: ChipProps['design'][] = ['outlined', 'filled', 'dashed', 'tinted']
+
+const intents: ChipProps['intent'][] = [
+  'primary',
+  'secondary',
+  'success',
+  'alert',
+  'danger',
+  'info',
+  'neutral',
+  'surface',
+]
+
 export default meta
 
 export const Default: StoryFn = _args => (
-  <Chip>
-    Hello World!
-    <Chip.ClearButton onClick={() => console.log('clear')} />
-  </Chip>
+  <div className="flex flex-col items-start gap-md">
+    {designs.map(design => (
+      <Chip design={design} key={design}>
+        <Chip.Content>Hello World!</Chip.Content>
+        <Chip.ClearButton onClick={() => console.log(`clear ${design}`)} />
+      </Chip>
+    ))}
```

Comment:
Perhaps we need a `design` story ? and a clear button story ?

Reply:
stories Pr are not created yet. This is just to test it working